src/mlbstandings/service.py

```
@@ -0,0 +1,37 @@
+        super().__init__()
+
+    def load_config(self):
+        config = {key: value for key, value in self.options.items()
+                  if key in self.cfg.settings and value is not None}
+        for key, value in config.items():
+            self.cfg.set(key.lower(), value)
+
+    def load(self):
+        return self.application
+
+def run_server(application, port: str, workers: int):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)